### PR TITLE
Add stone adze.

### PR DESCRIPTION
**Pull request by Rail-Runner**
Add stone adze and recipe to make it.

Took from Wilderness Overhaul mod by MormonPartyboat.
( http://smf.cataclysmdda.com/index.php?topic=9568 )

### CODE REVIEW

**Comment by Rivet-the-Zombie** [comment]
Very nice! Only thing I can think to add is that it should probably show up in museums as well.

**Comment by Darkyhard** [comment]
Done.

**Comment by PropaneSoup** [comment]
Lovin' these recipes Darkyhard

**Comment by KA101** [comment]
Sure, I'll merge this, though in future, mind putting 'em with the other stone tools, rather than at the end?  Thanks.

File: data/json/recipes/recipe_others.json [hunk 1]
```diff
@@ -6734,6 +6734,31 @@
...
+    "qualities":[
+      {"id":"HAMMER","level":2,"amount":1},
+      {"id":"CUT","level":1,"amount":1}
+    ],
+    "components": [
+      [ [ "stick", 1 ], 
+        [ "2x4", 1 ] ],
+      [ [ "rock", 1 ],
+        [ "ceramic_shard", 1 ],
+        [ "sharp_rock", 1 ] ],
```
Comment (KA101) [inline data/json/recipes/recipe_others.json]: Syntax issue: easier to read these if the brackets that open and close a group are on their own lines.  Not a blocker of itself, though, and I'll take care of it in the merge.

**Pull request merged**